authors/settings.py

```diff
@@ -21,7 +21,7 @@
 # See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '7pgozr2jn7zs_o%i8id6=rddie!*0f0qy3$oy$(8231i^4*@u3'
+SECRET_KEY = config('SECRET_KEY', '')
```

Comment:
@felixkiryowa @ja-odur wouldn't it be better to "hide" the secret key when the application is almost done. Mainly because I don't think the rest of the team has this environment variable set?

Reply:
This is resolved already actually, I had just forgotten changing it in the settings file. That's why it's reflecting those changes here in this PR